docker/data-science/notebooks/ch06/2.related-keywords-from-signals.ipynb

```diff
@@ -436,10 +440,10 @@
       "|        laptop|    computers|    2794|     349|      3143|        94|\n",
       "|       netbook|      laptops|    1018|    2781|      3799|        91|\n",
       "|    headphones|    headphone|    1617|     367|      1984|        90|\n",
-      "|        tablet|    computers|    1124|     449|      1573|        89|\n",
       "|        laptop|           hp|    2078|     749|      2827|        89|\n",
-      "|           mac|        apple|    1668|    1218|      2886|        88|\n",
+      "|        tablet|    computers|    1124|     449|      1573|        89|\n",
```

Comment:
Same issue from earlier. This sorting is apparently non-deterministic. Manuscript has previous sort order.